baseband/vlbi_base/base.py

```diff
@@ -189,18 +190,18 @@ def tell(self, unit=None):
             return self.start_time + self.tell(unit=u.s)
 
         return (self.offset * u_sample).to(unit, equivalencies=[(u.s, u.Unit(
-            self.samples_per_frame * self.frames_per_second * u_sample))])
+            self._sample_rate * u_sample))])
 
     def _frame_info(self):
         offset = (self.offset +
                   self.header0['frame_nr'] * self.samples_per_frame)
-        full_frame_nr, extra = divmod(offset, self.samples_per_frame)
-        dt, frame_nr = divmod(full_frame_nr, self.frames_per_second)
-        return int(dt), int(frame_nr), extra
+        dt, subsecond_samples = divmod(offset, self._sample_rate)
```

Comment:
One place where `_sample_rate` is functional. I'd still just do `self.sample_rate.to_value(u.Hz)` (faster than `to(u.Hz).value`, but may need newer astropy...)